### PR TITLE
Set-DbaAgentOperator - Fix usage of parameter Operator

### LINKED ISSUE #7730: Set-DbaAgentOperator when changing email at least changes all the operators not just the one specify
### Verified issue does not already exist?

No

### What error did you receive?

N/A

### Steps to Reproduce

```powershell
# provide your command(s) executed pertaining to dbatools
# please include variable values (redacted or fake if needed) for reference
```
Will change every operator, when I wrote this I only had one operator, so didn't text for multiple operators.
$server = "SQL01"
$OperatorName = "DBA"
$EmailAddress = "mail@mail.com"

Set-DbaAgentOperator -SQLInstance $server -Operator $operatorName -EmailAddress $EmailAddress

### Are you running the latest release?

Yes

### Other details or mentions

_No response_

### What PowerShell host was used when producing this error

VS Code (terminal), VS Code (integrated terminal)

### PowerShell Host Version

image

### SQL Server Edition and Build number

Microsoft SQL Server 2016 (SP2-CU17) (KB5001092) - 13.0.5888.11 (X64)   Mar 19 2021 19:41:38   Copyright (c) Microsoft Corporation  Enterprise Edition: Core-based Licensing (64-bit) on Windows Server 2016 Standard 10.0 <X64> (Build 14393: )

### PR DESCRIPTION
**Pull request by tboggiano**
## Type of Change

 - [X] Bug fix (non-breaking change, fixes #7730 )
 - [ ] New feature (non-breaking change, adds functionality, fixes # )
 - [ ] Breaking change (effects multiple commands or functionality, fixes # )
 - [ ] Ran manual Pester test and has passed (`.\tests\manual.pester.ps1`)
 - [ ] Adding code coverage to existing functionality
 - [ ] Pester test is included
 - [ ] If new file reference added for test, has is been added to github.com/sqlcollaborative/appveyor-lab ?
 - [ ] Unit test is included
 - [ ] Documentation
 - [ ] Build system

### Purpose

- Fix the fact this was updating all the the operators instead of just the ones specified.

### Commands to test

- Set-DbaAgentOperator -SQLInstance . -Operator DBA -EmailAddress Test@test.com2
- Set-DbaAgentOperator -SQLInstance . -Operator DBA, DBA2 -EmailAddress Test@test.com2
- 
### Screenshots

image

### CODE REVIEW

File: functions/Set-DbaAgentOperator.ps1 [hunk 1]
```diff
@@ -274,14 +274,13 @@ function Set-DbaAgentOperator {
...
             try {
-                $InputObject += Get-DbaAgentOperator -SqlInstance $SqlInstance -SqlCredential $SqlCredential -Operator $($op.Name) -EnableException
+                $InputObject += Get-DbaAgentOperator -SqlInstance $SqlInstance -SqlCredential $SqlCredential -Operator $Operator -EnableException
             } catch {
                 Stop-Function -Message "Failed" -Category ConnectionError -ErrorRecord $_ -Target $instance -Continue
             }
         }
 
         foreach ($op in $InputObject) {
-            $server = $op | Get-ConnectionParent
```
Comment (andreasjordan) [inline functions/Set-DbaAgentOperator.ps1:284]: You need $server in line 294 for the refresh of the SMO - why have you deleted this line?

**Comment by wsmelton** [comment]
We use the PR title for generating change log content now, please update the title to be more clear. As well, please link the issue this PR is related to or fixing so it will be closed properly.

File: functions/Set-DbaAgentOperator.ps1 (hunk 1 above)
Comment (andreasjordan) [reply functions/Set-DbaAgentOperator.ps1:284]: @potatoqualitee - Please add this line back in. Then this PR is ready for merge.

Comment (potatoqualitee) [reply functions/Set-DbaAgentOperator.ps1:284]: will do, thanks @andreasjordan !

**Comment by potatoqualitee** [comment]
🎈

**Pull request merged**